test/integration_test_wrapper.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python3
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+'''Test wrapper command for driving integration tests.
+
+Note: This is deliberately rough and only intended to drive existing tests
+with the expectation that as part of formally defining the API it will be tidy.
+
+'''
+
+from argparse import ArgumentParser
+from contextlib import ExitStack
+from logging import getLogger as logging_get_logger, basicConfig as logging_basic_config, DEBUG as LOG_DEBUG
+from pathlib import Path
+from shlex import quote
+from subprocess import DEVNULL, run
+
+
+log = logging_get_logger(__name__)
```

Comment:
I would drop this, just using `logging.info()`, etc should be good enough for a script.